generic/generic_font-1.2.3.recipe

```diff
@@ -24,16 +24,9 @@ REQUIRES=""
 BUILD_REQUIRES=""
 BUILD_PREREQUIRES=""
 
-BUILD()
-{
-	true
-}
-
 INSTALL()
 {
 	FONTDIR=$fontsDir/ttfonts
 	mkdir -p ${FONTDIR}
-	cp generic-font1.ttf ${FONTDIR}
-	cp generic-font2.ttf ${FONTDIR}
-	cp generic-font-bold.ttf ${FONTDIR}
+	cp -a *.ttf ${FONTDIR}
```

Comment:
@jmairboeck better this way?

Reply:
I think for the sample it doesn't really matter. LGTM.